source/Tutorials/Advanced/Security/Deployment-Guidelines.rst

```diff
@@ -0,0 +1,214 @@
+
+The following table depicts a summary of the previous statements relating the Keystore directory with the Recommended location:
+
++------------------------+--------------+---------------+---------------------+
+| Directory / Location   | Organization | Remote System | Material Sensitivity|
++========================+==============+===============+=====================+
+| public                 |       ✓      |       ✓       |         Low         |
++------------------------+--------------+---------------+---------------------+
+| private                |       ✓      |       ✕       |         High        |
++------------------------+--------------+---------------+---------------------+
+| enclaves               |       ✓      |       ✓       |        Medium       |
++------------------------+--------------+---------------+---------------------+
+
+
+Building a deployment scenario
+------------------------------
+
+To illustrate a simple deployment scenario, a docker image will be built intented to be the remote production system actor.
+In this example, localhost will serve as the organization's system.
+To test security capabilities, a secure listener will be launched on the remote system, whereas a secure talker will be launched on the localhost.
+Let's start by creating a workspace tree with the following sub-folders:
+
+.. code-block:: bash
+
+  mkdir -p ~/deploy_gd_tutorial/organization_system
+  mkdir ~/deploy_gd_tutorial/remote_system
+  mkdir ~/deploy_gd_tutorial/keystore
+
+Similarly to previous tutorials, intialize a new keystore tree directory.
+This will create enclaves/ public/ and private/ directories, which are explained in more detail in `ROS2 Security enclaves <https://design.ros2.org/articles/ros2_security_enclaves.html>`_.
+
+.. code-block:: bash
+
+  # Source ROS installation
+  source /opt/ros/${ROS_DISTRO}/setup.bash
+  # Initialize a new keystore directory
+  ros2 security create_keystore ~/deploy_gd_tutorial/keystore
+
+At this point, step into the remote_system workspace, create the corresponding enclave and copy just the public/ and enclaves/ directories to the current one.
+Those security artifacts will be needed by the remote system to enable listener's security.
+For the sake of simplicity, the same CA is used within this enclave for both, identity and permissions.
+Note that private/ folder is not moved but left in localhost (organization).
+
+.. code-block:: bash
+
+  # Move to remote's system path
+  cd ~/deploy_gd_tutorial/remote_system
+  # Create an enclave for the secure listener
+  ros2 security create_enclave ~/deploy_gd_tutorial/keystore /talker_listener/listener
+  # Ship enclaves/ and public/ directories only, minimizing security threat
+  cp -R ../keystore/enclaves ../keystore/public .
+
+For the purpose of running a secure listener at the docker image startup, a new ``entrypoint.sh`` file is required with the following content:
```

Comment:
Maybe this requires a subsection. It was not clear to me where to put the Dockerfile and entrypoint.sh files